packages/deprecated/http/httpcall_client.js

```diff
@@ -110,7 +112,7 @@ HTTP.call = function(method, url, options, callback) {
     else
       throw new Error("Can't create XMLHttpRequest"); // ???
 
-    xhr.open(method, url, true, username, password);
+    xhr.open(method, url, true);
```

Comment:
Good question, it was not passing the info to the server, so I had to change it to pass the tests. I was using latest chrome to test.

Reply:
It seems to have landed on chrome at 2021, after the package was moved to deprecated and, probably, not tested anymore.

Reply:
That's interesting.

The only issue with the new code would be if the password or username has any unicode characters since btoa throws for them: https://developer.mozilla.org/en-US/docs/Web/API/btoa#unicode_strings. I don't know how important that is. Besides that issue, the new implementation looks good to me.